cmake/OpenCVFindCANN.cmake

```diff
@@ -5,7 +5,19 @@ if("cann${CANN_INSTALL_DIR}" STREQUAL "cann" AND DEFINED ENV{ASCEND_TOOLKIT_HOME
     message(STATUS "CANN: updated CANN_INSTALL_DIR from ASCEND_TOOLKIT_HOME=$ENV{ASCEND_TOOLKIT_HOME}")
 endif()
 
+if(EXISTS "${CANN_INSTALL_DIR}/opp/op_proto/built-in/inc")
+    set(CANN_VERSION_BELOW_6_3_ALPHA002 "YES" )
+    add_definitions(-DCANN_VERSION_BELOW_6_3_ALPHA002="YES")
+endif()
+
 if(CANN_INSTALL_DIR)
+    # Supported system: UNIX
+    if(UNIX)
+    else()
+        set(HAVE_CANN OFF)
+        message(STATUS "CANN: CANN toolkit supports unix but not ${CMAKE_SYSTEM_NAME}. Turning off HAVE_CANN")
```

Comment:
I propose to use message(WARNING). It's highlighted red and caught by CI.

Reply:
I use message(WARNING) in every place that "set(HAVE_CANN OFF)".